lib/rules/declaration-block-single-line-max-declarations/index.js

```diff
@@ -48,7 +47,6 @@ const rule = (primary) => {
 			report({
 				message: messages.expected(primary),
 				node: ruleNode,
-				index: beforeBlockString(ruleNode, { noRawBefore: true }).length,
```

Comment:
[ask] The previous version seems more useful to me because the rule is about a block `{ }` (especially when a selector is long). Any thoughts?

(Screenshots on VS Code ⬇️ )

### Before

<img width="537" alt="image" src="https://user-images.githubusercontent.com/473530/157666065-5e875a27-86c8-4c07-b6d0-7fa59708a457.png">

### After

<img width="535" alt="image" src="https://user-images.githubusercontent.com/473530/157666143-17b25586-c752-4a45-b571-8a264e8861a8.png">

Reply:
I agree. Let's revert.

The `endIndex` should point to the closing brace.

> because the rule is about a block { } (especially when a selector is long). 

Yes, that's correct. The majority of the rules [are named](https://stylelint.io/user-guide/rules/about#names) so that the first part describes the scope of the rule, e.g. [`declaration-property-unit-allowed-list`](https://stylelint.io/user-guide/rules/list/declaration-property-unit-allowed-list) is named as such (where the scope is `declaration`) rather than named `property-unit-allowed-list`. I think we can generally follow this pattern when deciding where the `index` and `endIndex`s should be.

Reply:
@jeddy3 Thank you. I understand well. 👍🏼

Reply:
Fixed via 19253dd0